src/managers/channel-manager.js

```diff
@@ -171,7 +172,7 @@ ChannelManager.prototype = $.extend(ChannelManager.prototype, {
      *
      * **Example**
      *
-     *      var cm = new F.manager.ChannelManager();
+     *      var cm = new F.manager.ChannelManager({shareConnection: true});
```

Comment:
I think the default example should always be the simplest - `var cm = new F.manager.ChannelManager()`. If it turns out that shareConnection indeed does work we'll just make that the default.